lib/cog/commands/group.ex

```diff
@@ -27,12 +25,15 @@ defmodule Cog.Commands.Group do
   option "list", type: "bool"
 
   option "user", type: "string"
-  option "group", type: "string"
 
   permission "manage_groups"
   permission "manage_users"
 
-  rule "when command is #{Cog.embedded_bundle}:group must have #{Cog.embedded_bundle}:manage_groups"
+  rule "when command is #{Cog.embedded_bundle}:group with option[create] == true must have #{Cog.embedded_bundle}:manage_groups"
+  rule "when command is #{Cog.embedded_bundle}:group with option[drop] == true must have #{Cog.embedded_bundle}:manage_groups"
+  rule "when command is #{Cog.embedded_bundle}:group with option[list] == true must have #{Cog.embedded_bundle}:manage_groups"
+  rule "when command is #{Cog.embedded_bundle}:group with option[add] == true must have #{Cog.embedded_bundle}:manage_users"
```

Comment:
I believe the rationale from the long-long-ago was that by adding a user to a group, you're affecting the user more than you're affecting the group (e.g., giving them permissions they didn't have before)

@imbriaco Sound fair?